### PR TITLE
fix: social image url using gatsby image

### PR DESCRIPTION
**Pull request by raae**
Did the minimal changes to get this to work, hope it's helpful.

### CODE REVIEW

File: src/definitions/index.d.ts [hunk 1]
```diff
@@ -54,5 +55,6 @@ interface INode {
     title: string
     tags?: string
     description: string
+    banner: any
```
Comment (raae) [inline src/definitions/index.d.ts:58]: I took the easy way here and did any 🤣

File: src/templates/BlogPost.tsx [hunk 2]
```diff
@@ -96,7 +104,13 @@ export const pageQuery = graphql`
         title
         date(formatString: "MMMM DD, YYYY")
         description
-       
+        banner {
```
Comment (raae) [inline src/templates/BlogPost.tsx:107]: Get the sharp data

**Review by gtchakama — APPROVED** [review]

**Comment by raae** [comment]
If you want I can make another PR towards your repo, but didn't want to do that without asking @gtchakama!

**Comment by gtchakama** [comment]
Hahaha its okay, thank you for the help

**Comment by gtchakama** [comment]
I had managed to see the changes and implemented them.

**Pull request closed without merging**